fdbclient/DatabaseContext.h

```diff
@@ -364,7 +384,8 @@ class DatabaseContext : public ReferenceCounted<DatabaseContext>, public FastAll
 	                                   std::unique_ptr<SpecialKeyRangeReadImpl> &&impl);
 
 	static bool debugUseTags;
-	static const std::vector<std::string> debugTransactionTagChoices; 
+	static const std::vector<std::string> debugTransactionTagChoices;
+	std::map<Key, Reference<WatchMetadata>> watchMap;
```

Comment:
I think it would be good to make too changes here:

1. Have the `Key` in `WatchMetadata` and make the key a simple `StringRef`
2. Change `map` to `unordered_map`